app/views/portal/show.rb

```diff
@@ -4,10 +4,35 @@ class Show < ApplicationView
     attr_accessor :document, :debug
 
     def head_meta
+      desc = render_document_show_field_value(document, 'proxies.dcDescription', unescape: true)
+      landing = render_document_show_field_value(document, 'europeanaAggregation.edmLandingPage')
+      preview = render_document_show_field_value(document, 'europeanaAggregation.edmPreview')
+      title = render_document_show_field_value(document, 'proxies.dcTitle', unescape: true)
+
       mustache[:head_meta] ||= begin
         [
-          { meta_name: 'description', content: truncate(strip_tags(render_document_show_field_value(document, 'proxies.dcDescription')), length: 350, separator: ' ') }
-        ] + super
+          { meta_name: 'description', content: truncate(strip_tags(render_document_show_field_value(document, 'proxies.dcDescription')), length: 350, separator: ' ') },
+          { meta_name: 'twitter:card', content: 'summary' },
+          { meta_name: 'twitter:site', content: '@EuropeanaEU' },
+          begin
+            { meta_property: "og:title", content: title } unless title.nil?
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.